scripts/compose.py

```diff
@@ -107,7 +107,7 @@ def load_images(urls, cache_dir):
         sys.exit(1)
 
 
-DEFAULT_HEALTHCHECK_INTERVAL = "5s"
+DEFAULT_HEALTHCHECK_INTERVAL = "10s"
```

Comment:
increase the health check interval time to the Docker containers health check service, 10 seconds is better than 5